projects/go-style-guide/src/app/features/ui-kit/components/button-docs/button-docs.component.html

```diff
@@ -1,5 +1,6 @@
 <header class="go-page-title">
   <h1 class="go-heading-1">{{ pageTitle }}</h1>
+  <go-button (handleClick)="linkToSource()" buttonVariant="neutral"> Link To Source </go-button>
```

Comment:
@Ashygoyal what are your thoughts on changing these buttons to say "View on GitHub" instead?

Reply:
That sounds better. I'll change it up.